src/main/java/com/example/backendspringboottechiteasycontroller/controllers/TelevisionController.java

```diff
@@ -0,0 +1,64 @@
+package com.example.backendspringboottechiteasycontroller.controllers;
+
+import com.example.backendspringboottechiteasycontroller.exceptions.RecordNotFoundException;
+import com.example.backendspringboottechiteasycontroller.service.Television;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.List;
+
+
+@RestController
```

Comment:
Je kunt voor de netheid hier:
@RequestMapping("/television") toevoegen dan kun je "/television" weglaten in al je andere endpoints die je in deze controller aanmaakt